src/styles/styles.js

```diff
@@ -0,0 +1,34 @@
+export const styles = {
+  colors: {
+    gray1: "rgb(45, 45, 45)",
+    gray2: "rgb(70, 70, 70)",
+    blue1: "rgb(0, 136, 235)",
+    tBlue: "rgba(0, 136, 235, 0.75)",
+  },
+  sizes: {
+    thickness: {
+      thick1: 1,
+      thick2: 2,
+      thick3: 4,
+      thick4: 5,
+      thick5: 10,
+      thick6: 15,
+      thick7: 20,
+      thick8: 25,
+      thick9: 30,
+      thick10: 35,
+      thick11: 40,
+      thick12: 45,
+      thick13: 50
+    },
+    specialThick: { //This rule is for really specific numbers. This way, that values do not push the normal ones.
+      thick1: 42
```

Comment:
since this is a specialThick can you define a special name for it? so that we can know why or where it's used